CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.9)
 
-add_compile_options(-Wall -Wextra -Werror) #-pedantic
+add_compile_options(-Wall -Wextra) #-pedantic
```

Comment:
Not sure we should be removing errors from the output.
Would need to investigate fixing the code to be compliant with the compiler to 'properly' fix this.